content/en/aws/managed-streaming-for-kafka/index.md

```diff
@@ -66,7 +68,155 @@ The expected output is something like the following
+
+{{< command >}}
+{
+    "BootstrapBrokerString": "localhost:4514"
+}
+{{< / command >}}
+
+Run the following command in the bin folder, replacing `BootstrapBrokerStringTls` with the value that you obtained when you ran the previous command.
+
+{{< command >}}
+./kafka-console-producer.sh --broker-list BootstrapBrokerStringTls --producer.config client.properties --topic LocalMSKTopic
+{{< / command >}}
+
+Enter any message that you want, and press Enter. Repeat this step two or three times. Every time you enter a line and press Enter, that line is sent to your Apache Kafka cluster as a separate message.
+
+Keep the connection to the client machine open, and then open a second, separate connection to that machine in a new window.
+
+In the following command, replace ```BootstrapBrokerStringTls``` with the value that you saved earlier. Then, go to the bin folder and run the command using your second connection to the client machine.
+
+{{< command >}}
+./kafka-console-consumer.sh --bootstrap-server BootstrapBrokerStringTls --consumer.config client.properties --topic LocalMSKTopic --from-beginning
+{{< / command >}}
+
+You start seeing the messages you entered earlier when you used the console producer command. These messages are TLS encrypted in transit.
+
+Enter more messages in the producer window, and watch them appear in the consumer window.
+
+## Delete the local MSK cluster
+
+Run the following command to list your msk clusters
+
+{{< command >}}
+awslocal kafka list-clusters --region us-east-1
+{{< / command >}}
+
+From the list of clusters, pick the ```ClusterARN``` of the cluster you want deleted and run the command
+
+{{< command >}}
+awslocal kafka delete-cluster --region us-east-1 --cluster-arn ClusterArn
+{{< / command >}}
+
+## Local MSK and Lambdas
+
+### Adding a local MSK trigger 
+
+The following example uses the create-event-source-mapping awslocal command to map a Lambda function named my-kafka-function to a Kafka topic named LocalMSKTopic. The topic's starting position is set to LATEST.
+
+{{< command >}}
+awslocal lambda create-event-source-mapping \
+  --event-source-arn arn:aws:kafka:us-east-1:000000000000:cluster/EventsCluster \
+  --topics LocalMSKTopic \
+  --starting-position LATEST \
+  --function-name my-kafka-function
+{{< / command >}}
+
+The following answer is to be expected
+
+{{< command >}}
+{
+    "UUID": "9c353a2b-bc1a-48b5-95a6-04baf67f01e4",
+    "StartingPosition": "LATEST",
+    "BatchSize": 100,
+    "ParallelizationFactor": 1,
+    "EventSourceArn": "arn:aws:kafka:us-east-1:000000000000:cluster/EventsCluster",
+    "FunctionArn": "arn:aws:lambda:us-east-1:000000000000:function:my-kafka-function",
+    "LastModified": "2021-11-21T20:55:49.438914+01:00",
+    "LastProcessingResult": "OK",
+    "State": "Enabled",
+    "StateTransitionReason": "User action",
+    "Topics": [
+        "LocalMSKTopic"
+    ]
+}
+{{< / command >}}
+
+
+### Viewing the status using the awslocal
```

Comment:
I think we can remove this subsection. 

Instead, I'd add a short sentence to like:

> Using this event source mapping, LocalStack will automatically spawn Lambda functions for each message that gets published to the target Kafka topic. You can use the `kafka-console-producer.sh` client script (see above) to publish messages to the topic, and then observe the LocalStack log output to see how Lambda function are executed (in Docker containers) as new messages arrive.

Reply:
Done